pkg/exchange/bitget/exchange.go

```diff
@@ -116,8 +116,21 @@ func (e *Exchange) QueryKLines(ctx context.Context, symbol string, interval type
 }
 
 func (e *Exchange) QueryAccount(ctx context.Context) (*types.Account, error) {
-	// TODO implement me
-	panic("implement me")
+	req := e.client.NewGetAccountAssetsRequest()
+	resp, err := req.Do(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	bals := types.BalanceMap{}
+	for _, asset := range resp {
+		b := toGlobalBalance(asset)
+		bals[asset.CoinName] = b
+	}
```

Comment:
Perhaps you can move this implementation to QueryAccountBalances, and then QueryAccount can utilize it.